parm/config/gefs/config.base

```diff
@@ -133,11 +133,8 @@ export DO_COUPLED="NO"
 export DO_WAVE="NO"
 export DO_OCN="NO"
 export DO_ICE="NO"
-export DO_AERO="NO"
```

Comment:
Why is there no `DO_AERO` for GEFS anymore (and not exported for GFS)?

Reply:
Because aerosols are controlled at the forecast and analysis level, not the application level.

For instance, previously, this was implemented in the `config.fcst` scripts by reading the `AERO_FCST_RUN` variable:
https://github.com/NOAA-EMC/global-workflow/blob/152bb45041ff7b69d14db98648b351a8f527e8d5/parm/config/gfs/config.fcst#L11-L21

But now, it can rely on just the `DO_AERO_FCST` variable.

Keeping the `DO_AERO` flag would be redundant since it is superseded by `DO_AERO_FCST` (or `DO_AERO_ANL` in the analysis jobs).

Reply:
I debated whether to add `DO_AERO` to `config.gefs`.  When analyses are enabled, having the flag will be handy.